impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -76,9 +77,13 @@ public class KafkaClusterConfig {
 
     /**
      * Kafka version to be used for deploying kafka in container mode, e.g. "3.3.1".
+     * Defaults to the version available on the classpath.
+     * <br/>
+     * This value is ignored if execMode is not CONTAINER.
```

Comment:
I think the double negative is a confusing, personally I'd say something to the effect 

`The value is only used when execMode is CONTAINER`